### ISSUE TITLE
Slower execution of bun install compared to yarn add for the same library on Windows WSL

### ISSUE THREAD

**Issue by marcoshenrique-dev** [oven-sh/bun#6212]
### What version of Bun is running?

1.0.3+25e69c71e70ac8a0a88f9cf15b4057bd7b2a633a

### What platform is your computer?

Linux 5.15.90.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

Run bun install express or bun add express (I tested it with express and @types/express)

### What is the expected behavior?

It was expected to run 33x faster than yarn, according to the bun's own website

### What do you see instead?

![image](https://github.com/oven-sh/bun/assets/51785898/73181af3-9d8c-4819-beb6-063b3ce58f9e)
Bun add took 400 seconds (6 minutes) to install @types/express (and four minutes to add express)
![image](https://github.com/oven-sh/bun/assets/51785898/4db6b2d9-c2a7-4278-b8fb-f5a8a35cda31)
While yarn add @types/express took only 13 seconds to finish the install (and 3.33 seconds to add express)
![image](https://github.com/oven-sh/bun/assets/51785898/e58cd6e5-1923-42d9-bc99-fab0f893d26f)
Internet status

### Additional information

This problem may be related to the execution within WSL or even to the support of the packages. Thanks for your attention.

**Comment by ATQQ** [comment]
Is it possible that the `npm registry` is different？

**Comment by marcoshenrique-dev** [comment]
I don't think so, I've seen people commenting that it worked well on Linux/Mac, I'll do some tests on Ubuntu to see if there's a difference

**Comment by roiiiu** [comment]
same here.

**Comment by zavx0z** [comment]
same here.

**Comment by Firgrep** [comment]
Took me 300 secs to install ~850 packages on a NestJS project and roughly 60 secs to run the darn thing. Running Ubuntu in WSL.
